srv/GetCartesianPath.srv

```diff
@@ -43,6 +43,13 @@ bool avoid_collisions
 # Specify additional constraints to be met by the Cartesian path
 Constraints path_constraints
 
+# Maximum cartesian speed for the given link.
+# If max_cartesian_speed <= 0 the trajectory is not modified.
+# WARNING NOT FUNCTIONAL YET.
```

Comment:
You can remove these comments from the commits and mark your PR as dependent or WIP instead.